### DISCUSSION TITLE
November 2023 product updates

### DISCUSSION THREAD

**Discussion by elizafarley** [amzn/ads-advanced-tools-docs#183] [Announcements]
This discussion will highlight releases and product updates for November 2023. If you would like to be notified when we post updates, you can subscribe to this discussion. Feel free to ask questions or leave feedback in the thread!

**Comment by elizafarley** [comment]
**Amazon DSP launches goal-based bidding**

Amazon DSP is launching goal-based bidding, which helps advertisers to specify a performance target for their campaigns and allow Amazon DSP to automatically optimize their bids in real-time in order to maximize delivery at their performance target. With this release, advertisers can automatically optimize to both delivery and performance goals.

Goal-based bidding is part of the ‘optimization’ field of the order object in [POST /dsp/orders](https://advertising.amazon.com/API/docs/en-us/dsp-campaigns#tag/Order/operation/CreateOrders). To get suggested KPI target range, you can use the [POST /dsp/campaigns/targetKpi/recommendations](https://advertising.amazon.com/API/docs/en-us/target-kpi-recommendations#operation/getGsbTargetKpiRecommendation) endpoint.

**Comment by elizafarley** [comment]
**DSP audiences edit API now supports editing the name, ASIN list, and lookback period for product audiences**

Advertisers can now edit the name, ASIN list, and lookback period for product audiences that are not part of an active campaign.

For technical specifications, please consult the [API reference](https://advertising.amazon.com/API/docs/en-us/audiences#tag/Ads/operation/DspAudienceEdit).

**Comment by elizafarley** [comment]
**Sponsored Brands version 2 reporting now supports click and view-based attribution**

Sponsored Brands reporting version 2 reports now support view and click attributed data for conversion metrics (that is, conversions, sales, units sold, new-to-brand, and detail-page-view metrics) for Sponsored Brands campaigns with a costType of VCPM. The conversion metrics will have the right data depending on the attribution type for the campaign. For campaigns with a costType of CPC, metrics will continue to report click attributed data. 

See [version 2 metrics](https://advertising.amazon.com/API/docs/en-us/guides/reporting/v2/metrics) for updated metric definitions.

**Comment by elizafarley** [comment]
**Sponsored Display updates to targeting recommendation API**

Sponsored Display now includes themes in the response of the [targeting recommendations API](https://advertising.amazon.com/API/docs/en-us/sponsored-display/3-0/openapi#tag/Targeting-Recommendations).

Theme-based recommendations offer multiple options to target relevant products without the need for additional research from advertisers. Target products under each theme are curated specific to the use case, with high-quality suggestions based on Amazon's first-party signals. Each theme will have up to 50 individually recommended products, with flexibility to choose products individually or select all products within a theme.

For full technical details, please see our updated [targeting recommendations API](https://advertising.amazon.com/API/docs/en-us/sponsored-display/3-0/openapi#tag/Targeting-Recommendations) documentation.

**Comment by elizafarley** [comment]
**Sponsored Display allows optimization rules to be included in the forecasting API**

Sponsored Display has improved the accuracy of the [forecasting API](https://advertising.amazon.com/API/docs/en-us/sponsored-display/3-0/openapi#tag/Forecasts) by allowing [optimization rules](https://advertising.amazon.com/API/docs/en-us/sponsored-display/3-0/openapi#tag/Optimization-Rules-(beta)) to be included as an input.

Forecasting helps advertisers understand the impact of their choices during campaign creation. This includes a minimum and maximum range of available impressions, viewable impressions, and clicks that could be available based on the selected advertised products, optimization type, targeting, bid, and optimization rule.

For full technical details, please see our updated [forecasting API](https://advertising.amazon.com/API/docs/en-us/sponsored-display/3-0/openapi#tag/Forecasts).

**Comment by jerirose** [comment]
#### Translations of advanced tools center content now available in Chinese and Japanese

Developers can now read translations of advanced tools center content in Chinese and Japanese. The current translations reflect the status of advanced tools center documents as of July 2023.

Visit the [translations page in the advanced tools center](https://advertising.amazon.com/API/docs/en-us/guides/translations) for more information, or go directly to the [Chinese translation](https://d3a0d0y2hgofx6.cloudfront.net/zh-cn/index.html) or [Japanese translation](https://d3a0d0y2hgofx6.cloudfront.net/ja-jp/index.html).
